device/delta/x86_64-delta_et-6248brb-r0/Delta-et-6248brb/sai.profile

```diff
@@ -1 +1 @@
-SAI_INIT_CONFIG_FILE=/etc/bcm/helix4-et-6248brb-48x1G+2x10G.config.bcm
+SAI_INIT_CONFIG_FILE=/usr/share/sonic/platform/Delta-et-6248brb/helix4-et-6248brb-48x1G+2x10G.config.bcm
```

Comment:
should be /usr/share/sonic/hwsku/helix4-et-6248brb-48x1G+2x10G.config.bcm

Reply:
I have modified this sai.profile with correct configuration path